pkgs/tools/misc/fetch-scm/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, lib, fetchFromGitHub, guile }:
+
+let
+  file = "fetch.scm";
+  pkg = "fetch-scm";
```

Comment:
please inline this and make sure the rest of the file doesn't have extra indentation